exercises/roman-numerals/introduction.md

```diff
@@ -0,0 +1,43 @@
+# Description
+
+The Romans were a clever bunch.
+They conquered most of Europe and ruled it for hundreds of years.
+They invented concrete and straight roads and even central heating.
+But one thing they never discovered though was the number zero.
+
+Their numbering system, known today as "Roman Numerals" uses letters - I, V, X, L, C, D, M.
+
+| I | V | X  | L  |  C  |  D  |  M   |
+|---|---|----|----|-----|-----|------|
+| 1 | 5 | 10 | 50 | 100 | 500 | 1000 |
+
+There are four basic rules for Roman Numerals:
+
+- When a larger letter precedes a smaller letter, the letters are added together. For example: in `VI`, `V` (5) is larger than `I` (1), so we add them together to get 6.
```

Comment:
```suggestion
- When a larger letter precedes a smaller letter, the letters are added together.
For example: in `VI`, `V` (5) is larger than `I` (1), so we add them together to get 6.
```

One line per sentence or would this mess up the formatting?

Reply:
It will not mess up formatting. Indentation is allowed:

```suggestion
- When a larger letter precedes a smaller letter, the letters are added together.
  For example: in `VI`, `V` (5) is larger than `I` (1), so we add them together to get 6.
```